src/Form/Control/Password.php

```diff
@@ -4,7 +4,44 @@
 
 namespace Atk4\Ui\Form\Control;
 
+use Atk4\Ui\Button;
+use Atk4\Ui\Js\JsExpression;
+
 class Password extends Line
```

Comment:
Based on https://github.com/whatwg/html/issues/7293 it is true that native solution does not exist in all browsers currently.

at least https://stackoverflow.com/questions/61449079/how-to-hide-the-eye-from-a-password-input-in-ms-edge-and-ie#61450596 is needed to supress the native behaviour of IE, or do we want to keep native eye where supported?

Reply:
I think I meant a more native ATK4 solution with more Js functions instead of a big javascript blob... :)

Reply:
IE - I tested Windows 7 and 10, it seems Microsoft does everything to kill IE in favor of Edge - in both systems, when I opened IE, Edge was opened instead. Edge is based on Chrome. And in Edge and Chrome there is no native reveal icon added. So we do not need any additional code to coexist with the native reveal.